autolink/postlink/gradleLinker.js

```diff
@@ -1,60 +1,98 @@
 // @ts-check
-var path = require('./path');
-var fs = require('fs');
-var { warnn, errorn, logn, infon, debugn } = require('./log');
-var { insertString } = require('./stringUtils');
-var DEFAULT_KOTLIN_VERSION = '1.3.61';
+var path = require('./path')
+var fs = require('fs')
+var { warnn, errorn, logn, infon, debugn } = require('./log')
+var { insertString } = require('./stringUtils')
+var DEFAULT_KOTLIN_VERSION = '1.3.61'
 // This should be the minSdkVersion required for RNN.
 var DEFAULT_MIN_SDK_VERSION = 19
 
 class GradleLinker {
   constructor() {
-    this.gradlePath = path.rootGradle;
+    this.gradlePath = path.rootGradle
+    this.setKlotinVersionSuccess = false
+    this.setKotlinPluginDependency = false
```

Comment:
Same here: some variables has `Success` as a suffix, some others don't.
Not sure if it's intentional.

Reply:
It's not, I will use a standard naming, appending Success to all.